src/libraries/Common/src/Interop/OSX/NSRange.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using System.Runtime.InteropServices;
+namespace System.Globalization
```

Comment:
```suggestion
internal static partial class Interop
```
This type is not specific to globalization. It should have neutral namespace.

Reply:
I see the unmanaged side is not returning the NSRange type in the latest iteration. It returns its own Range type. In that case, it should not be called NSRange. It should be called Range to match the unmanaged side and it should be in System.Globalization.iOS.cs.

Reply:
I'm using `NSRange`  https://developer.apple.com/documentation/foundation/nsrange?language=objc on pal_collation.m,
but to make it compiled on https://github.com/dotnet/runtime/blob/main/src/native/libs/System.Globalization.Native/pal_collation.h I needed to add Range type.
Do you mean `Range` struct should be added in System.Globalization.iOS.cs ? I couldn't find that file

Reply:
I meant src/libraries/Common/src/Interop/Interop.Collation.OSX.cs